internal/core/src/mmap/Column.h

```diff
@@ -56,7 +54,7 @@ class ColumnBase {
         data_ = static_cast<char*>(mmap(nullptr,
                                         cap_size_ + padding_,
                                         PROT_READ | PROT_WRITE,
-                                        mmap_flags | MAP_ANON,
+                                        MAP_PRIVATE | MAP_ANON,
```

Comment:
ooc, why are we switching to map_private? What's the benefit?

Reply:
cmiiw, it's for the purpose of accurate memory count. Is there a performance impact though?

Reply:
For only accurate memory usage count, no performance impact.

Sure the private mapping would cause copy-on-write if some child thread writes the area, but we only write the mapping by the creator thread while loading, so it works well